### DISCUSSION TITLE
同步用户数据定时任务

### DISCUSSION THREAD

**Discussion by henry-proj** [rime/home#1692] [Ideas]
同步用户数据能否添加一个定时任务功能。例如设置为5分钟自动同步一次。

**Comment by TwoBirdsOnTheTree** [comment]
+1, 在多设备情况下很有帮助

Linux下应该可以通过crontab实现:
``` shell
*/5 * * * * cd ~/.local/share/fcitx5/rime && /usr/bin/rime_dict_manager -s
```
Windows下不知道怎么办

**Reply by obarisk** [reply]
windows 用 task manager

**Reply by TwoBirdsOnTheTree** [reply]
你用的效果怎么样?
感觉目前Rime不太支持这样做, 因为
-  Linux (fcitx5-rime)下如果有应用正在输入是无法同步的, 提示: `opening db userdb/LOCK Resource temporarily unavailable`
- Windows下点击同步也会弹出提示`小狼毫维护中`
